test/parallel/feeLock.API.feePayment.parallel.test.ts

```diff
@@ -235,9 +237,11 @@ test("gasless- For low-value swaps, token reservation status and pallet storage
       await api.query.feeLock.accountFeeLockData(testUser1.keyRingPair.address)
     )
   );
-  const waitingBlock = accountFeeLockData.lastFeeLockBlock + periodLength;
+  const waitingBlock = await stringToBN(
```

Comment:
why awaiting a function that is not async?

Reply:
changed